Tests/SubprocessTests/SubprocessTests+Windows.swift

```diff
@@ -456,7 +456,9 @@ extension SubprocessWindowsTests {
 
 // MARK: - PlatformOption Tests
 extension SubprocessWindowsTests {
-    @Test(.enabled(if: SubprocessWindowsTests.hasAdminPrivileges()))
+    // Disabled until we investigate CI specific failures
+    // https://github.com/swiftlang/swift-subprocess/issues/128
+    @Test(.enabled(if: false))
```

Comment:
nit: there's a `disabled` API which accepts a reason string, as well as a bugtracker URL API, which you could use like this:

```suggestion
    @Test(.disabled("Disabled until we investigate CI specific failures"), .bug("https://github.com/swiftlang/swift-subprocess/issues/128"))
```